Lib/test/test_argparse.py

```diff
@@ -2816,6 +2816,36 @@ def test_interleaved_groups(self):
         result = parser.parse_args('1 2 3 4'.split())
         self.assertEqual(expected, result)
 
+class TestGroupConstructor(TestCase):
+    def test_group_prefix_chars(self):
+        parser = ErrorRaisingArgumentParser()
+
+        msg = (
+            "The use of the undocumented 'prefix_chars' parameter in "
+            "ArgumentParser.add_argument_group is deprecated."
+        )
+        with self.assertWarnsRegex(DeprecationWarning, msg):
+            parser.add_argument_group(prefix_chars='-+')
```

Comment:
Test also `stacklevel`. The simplest way to do this is to check that `filename` refers to the current file:

```suggestion
        with self.assertWarnsRegex(DeprecationWarning, msg) as cm:
            parser.add_argument_group(prefix_chars='-+')
        self.assertEqual(cm.filename, __file__)
```